centreon/www/front_src/src/Resources/Actions/Resource/Check/ListCheckOptions.tsx

```diff
@@ -0,0 +1,169 @@
+import { makeStyles } from 'tss-react/mui';
+
+import IconCheck from '@mui/icons-material/CheckOutlined';
+import List from '@mui/material/List';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Popover from '@mui/material/Popover';
+import Divider from '@mui/material/Divider';
+
+import {
+  labelCheck,
+  labelCheckDescription,
+  labelForcedCheck,
+  labelForcedCheckDescription
+} from '../../../translatedLabels';
+
+import Text from './Text';
+
+const useStyles = makeStyles()((theme) => ({
+  button: {
+    alignItems: 'flex-start'
+  },
+  container: {
+    backgroundColor: theme.palette.background.default,
+    display: 'flex',
+    flexDirection: 'column',
+    minWidth: theme.spacing(13.5),
+    padding: theme.spacing(0, 1)
+  },
+  icon: {
+    minWidth: theme.spacing(3)
+  },
+  itemText: {
+    '& .MuiTypography-root': {
+      width: '100%'
+    },
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    margin: 0,
+    paddingRight: theme.spacing(0.5)
+  }
```

Comment:
we can be moire specific here about what the element is really referring to

```suggestion
  checkButton: {
    alignItems: 'flex-start'
  },
  container: {
    backgroundColor: theme.palette.background.default,
    display: 'flex',
    flexDirection: 'column',
    minWidth: theme.spacing(13.5),
    padding: theme.spacing(0, 1)
  },
  icon: {
    minWidth: theme.spacing(3)
  },
  checkMethodItem: {
    '& .MuiTypography-root': {
      width: '100%'
    },
    display: 'flex',
    flexDirection: 'row-reverse',
    margin: 0,
    paddingRight: theme.spacing(0.5)
  }
```

Reply:
I put it generic because button can be check button or forced check button 🤔